include/xgboost/c_api.h

```diff
@@ -145,55 +108,27 @@ XGB_DLL int XGBCreateEnclave(const char *enclave_image, int log_verbosity);
 /*!
  * \brief load a data matrix
  * \param fname the name of the file
+ * \param username the name of the file owner
  * \param silent whether print messages during loading
  * \param out a loaded data matrix
  * \return 0 when success, -1 when failure happens
  */
 XGB_DLL int XGDMatrixCreateFromFile(const char *fname,
+                                    char* username,
```

Comment:
Do we now support loading from unencrypted files?

Reply:
Yes!

Reply:
Awesome!

Reply:
Although, the Python API still needs to be fixed (along with the RPC stuff).